jobApplicationTracker/services/IJobApplicationHistoryService.cs

```diff
@@ -0,0 +1,17 @@
+using jobApplicationTrackerApi.DataModels;
+
+namespace jobApplicationTrackerApi.services;
+
+public interface IJobApplicationHistoryService
```

Comment:
Does it make sense to separate JobApplicationHistoryService from JobApplicationService ?

Reply:
Honestly, I wasn't sure whether to add it or not so I just added it anyway